### ISSUE TITLE
ButtonMenu: a11y test timing out causing builds to fail

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#3635]
## What will this achieve?

There is an a11y test in `ButtonMenu` that has been timing out intermittently causing PR checks to fail. This should most likely be skipped for now, especially since Carbon now has the `MenuButton`.

![image](https://github.com/carbon-design-system/ibm-products/assets/10215203/b8b2704b-5916-48de-9fae-fb230b7c1a5b)
